tests/data_utils_tests/data_tests.py

```diff
@@ -86,7 +87,13 @@ def test_multiply_transform(self):
         self.assertListEqual(list(result["B"]), ["a", "b", "c", "a", "b", "c"])
         self.assertEqual(len(result["A"]), len(self.df) * n_repeats)
 
-
+    def test_majorityvote_transform(self):
+        df1 = pd.DataFrame({"ID": [1,1,1,2,2,2], "model_output": [100,100,99,5,4,1]})
```

Comment:
1. include a test case with NaNs
2. please change the default "ID" to "data_point_id" which is reserved in Eureka and is universal to all datasets